zellij-tile/src/data.rs

```diff
@@ -67,6 +67,64 @@ impl Default for InputMode {
     }
 }
 
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq, Eq, Hash)]
+pub enum Theme {
+    Light,
+    Dark,
+}
+
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq, Eq, Hash)]
+pub enum PaletteSource {
+    Default,
+    Xresources,
+}
+pub mod colors {
```

Comment:
Out of curiosity, why do we need this when we have palettes? Is this used within plugins?

Reply:
This seems to be for the `Default` implementation? See the below comment :)

Reply:
it seems that it got lucky and survived when I was cleaning stuff up